src/wallet/asyncrpcoperation_common.cpp

```diff
@@ -202,10 +202,13 @@ void ThrowInputSelectionError(
             switch (err.side) {
                 case ActionSide::Input:
                     side = "inputs";
+                    break;
                 case ActionSide::Output:
                     side = "outputs";
+                    break;
                 case ActionSide::Both:
                     side = "actions";
+                    break;
```

Comment:
This appears to be a bugfix; should it be done separately?

Reply:
It is a bugfix, and I do have it fixed in another PR (that currently has some failing tests). But it’s in here because `-Wimplicit-fallthrough` caught it.

I can pull it into a separate small bugfix commit if that’s better.